### PR TITLE
Feature/5/create booking

### PR DESCRIPTION
**Pull request by JElgar**

### CODE REVIEW

File: uobtheatre/bookings/test/test_serializers.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                "consession_type_id": consession_type.id,
+            },
+        ],
+    }
+
+    serialized_booking = CreateBookingSerialiser(data=data, context={"user": user})
+    assert not serialized_booking.is_valid()
+
+    # Check consession type is not required (default to adult)
+    # TODO write test to check default to adult
```
Comment (JElgar) [inline uobtheatre/bookings/test/test_serializers.py:104]: Don't forget :eyes:

Comment (JElgar) [reply uobtheatre/bookings/test/test_serializers.py]: #28

**Review by streth11 — COMMENTED** [review]
LGTM

**Review by streth11 — APPROVED** [review]
looks better to me

**Pull request merged**